crates/pixi_build_frontend/src/tool/spec.rs

```diff
@@ -48,6 +60,82 @@ impl IsolatedToolSpec {
             ..self
         }
     }
+
+    /// Installed the tool in the isolated environment.
+    pub async fn install(
```

Comment:
Could we also report progress, I think we already have some `Reporter` in place.

Reply:
Lets do this in a different PR.